ovirtlago/__init__.py

```diff
@@ -133,6 +133,17 @@ def prepare_repo(
                         reposync_yum_config,
                         repos,
                     )
+        elif custom_sources:
```

Comment:
this comment also applies to the above reposync_yum_config check (not patched here)